toqito/state_props/is_abs_ppt.py

```diff
@@ -78,7 +74,10 @@ def _abs_ppt_constraints(eigvals: np.ndarray, dim: list[int]) -> list[np.ndarray
 def is_abs_ppt(mat: np.ndarray, dim: None | int | list[int] = None, tol: float = 1e-8) -> bool | int:
     r"""Determine whether a density matrix is absolutely PPT (PPT from spectrum).
 
-    A density matrix is absolutely PPT if every unitary conjugation of it is PPT. This is a spectrum-based property and can be checked using the Hildebrand/QETLAB constraints for small dimensions.
+    :cite:`Hildebrand_2007_PPT, QETLAB_link`.
+
+    A density matrix is absolutely PPT if every unitary conjugation of it is PPT. This is a spectrum-based
+    property and can be checked using the Hildebrand/QETLAB constraints for small dimensions.
```

Comment:
The constraints are from Hildebrand's paper, but the reference implementation is from QETLAB. These should be referred to as separate items.